include/openenclave/bits/sgx/sgxtypes.h

```diff
@@ -604,16 +607,22 @@ typedef struct _sgx_report_body
     uint8_t mrsigner[OE_SHA256_SIZE];
 
     /* (160) */
-    uint8_t reserved3[96];
+    uint8_t reserved3[32];
+
+    /* (192) CONFIGID */
+    uint8_t config_id[64];
 
     /* (256) Enclave product ID */
     uint16_t isvprodid;
 
     /* (258) Enclave security version */
     uint16_t isvsvn;
 
-    /* (260) Reserved */
-    uint8_t reserved4[44];
+    /* (260) CONFIGSVN */
+    uint16_t config_svn;
```

Comment:
Hi Alvin, PR #3723 was just merged which added `config_id` and `config_svn` to the `sgx_report_body_t` struct. The change was same as what you made here but comments are bit different that might trigger conflicts. Could you fetch the master and rebase your PR on top of that?

Reply:
Thanks for you notice! Yes you already fill up this gap between sgxsdk and oesdk. Let me fix the conflict.